app/src/main/java/com/example/bucketbuds/PageFragment.java

```diff
@@ -0,0 +1,183 @@
+package com.example.bucketbuds;
+
+import android.app.Activity;
+import android.content.Context;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.inputmethod.InputMethodManager;
+import android.widget.Toast;
+
+import androidx.appcompat.widget.SearchView;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.parse.FindCallback;
+import com.parse.Parse;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+import com.parse.ParseUser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import okhttp3.Headers;
+
+public class PageFragment extends Fragment {
```

Comment:
Could this class have a more descriptive name?

Reply:
yes and I will change it's corresponding PageFragmentAdapter class name